exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -220,17 +226,17 @@ func (e *exporter) sendBatch(ctx context.Context, lines []string) error {
 			return nil
 		}
 
-		e.settings.Logger.Debug(fmt.Sprintf("Accepted %d lines", responseBody.Ok))
-		e.settings.Logger.Error(fmt.Sprintf("Rejected %d lines", responseBody.Invalid))
+		e.settings.Logger.Sugar().Debugf("Accepted %d lines", responseBody.Ok)
+		e.settings.Logger.Sugar().Errorf("Rejected %d lines", responseBody.Invalid)
```

Comment:
Optional: 

Ideally, since these are both In the error path, they could either be one log line or the same log level.